sympy/physics/mechanics/body.py

```diff
@@ -204,12 +214,19 @@ def kinetic_energy(self, frame):
         Examples
         ========
 
+        As Body has been deprecated, the following examples are for illustrative
+        purposes only. The functionality of Body is fully captured by RigidBody
+        and Particle. To ignore the deprecation warning we can use the
+        ignore_warnings context manager.
```

Comment:
It would be better to use `:class:RigidBody` so that the html docs link through to the docs for the non-deprecated classes.

Reply:
`Body` has been removed from the online documentation, so this would not really matter. I can add `Body` and `JointsMethod` back to the online documentation. One of the reasons why we believed we could deprecate these two classes was that they were not being used a lot yet.

Reply:
I decided to readd `Body` and `JointsMethod` to the online documentation under a special `Deprecated Classes` tab.